packages/subgraph/tests/dao/dao.test.ts

```diff
@@ -474,6 +485,125 @@ describe('handleExecuted', () => {
     clearStore();
   });
 
+  test('successfuly creates action and proposal if not found', () => {
+    let tuple: Array<ethereum.Value> = [ethereum.Value.fromString('')];
+    let selector = '0x11111111';
+
+    let execResults = [
+      Bytes.fromHexString('0x11'),
+      Bytes.fromHexString('0x22')
+    ];
+
+    let failureMap = '2';
+
+    let event = createExecutedEvent(
+      [tuple, tuple],
+      [selector, selector],
+      false,
+      execResults,
+      failureMap
+    );
+
+    handleExecuted(event);
+
+    let proposalId = event.params.actor
+      .toHexString()
+      .concat('_')
+      .concat(event.params.callId.toHexString())
+      .concat('_')
+      .concat(event.transaction.hash.toHexString())
+      .concat('_')
+      .concat(event.transactionLogIndex.toHexString());
+
+    assert.entityCount('TransactionActionsProposal', 1);
+    assert.entityCount('Action', 2);
+
+    eq('TransactionActionsProposal', proposalId, 'id', proposalId);
+    eq('TransactionActionsProposal', proposalId, 'failureMap', failureMap);
+
+    for (let i = 0; i < event.params.actions.length; i++) {
+      let actionId = proposalId.concat('_').concat(i.toString());
+
+      eq('Action', actionId, 'id', actionId);
+      eq('Action', actionId, 'execResult', execResults[i].toHexString());
+      eq('Action', actionId, 'dao', DAO_ADDRESS);
+      eq('Action', actionId, 'proposal', proposalId);
+      eq(
+        'Action',
+        actionId,
+        'data',
+        encodeWithFunctionSelector(tuple, selector).toHexString()
+      );
+    }
+    assert.entityCount('Action', 2);
+  });
+
+  test('successfuly updates action and proposal if found', () => {
+    let tuple: Array<ethereum.Value> = [ethereum.Value.fromString('')];
+    let selector = '0x11111111';
+    let execResult = Bytes.fromHexString('0x11');
+    let failureMap = '2';
```

Comment:
perhpas we can make those global so we use them in miltiple tests

Reply:
Global variables wouldn't make much sense in this case. Thats why having variables in `describe` blocks are for, but it doesn't support.